crate/src/lib.rs

```diff
@@ -22,7 +22,7 @@ fn local_addr(port: u16) -> String {
 
 fn bin_url() -> String {
     format!(
-        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{}/master/2c9375ee5ee307c2ce870c7dbd25eefd84fe8c36/near-sandbox.tar.gz",
+        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{}/1.25.0/9b3d6ba551f561a028f0216051e031bc2ba0c6b7/near-sandbox.tar.gz",
```

Comment:
we don't need to change the branch as long as the commit is the start of the version. This is nicer since we can just supply the commit hash without the branch string later on when we want to be able to configure it from outside the library
```suggestion
        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{}/master/9b3d6ba551f561a028f0216051e031bc2ba0c6b7/near-sandbox.tar.gz",
```

Reply:
@ChaoticTempest That link (with "`/master/`") doesn't exist. This is the reason of having `1.25.0` in the URL.

Try it yourself:
1. DOESN'T WORK: https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/Linux-x86_64/master/9b3d6ba551f561a028f0216051e031bc2ba0c6b7/near-sandbox.tar.gz
2. WORK: https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/Linux-x86_64/1.25.0/9b3d6ba551f561a028f0216051e031bc2ba0c6b7/near-sandbox.tar.gz

I think that commit doesn't exist in the `master`, instead the branch is tagged, or something like this.

Reply:
ahh, I see. Ok, once you pull in the latest changes, you can just make these simples changes to the rust side and we should be good to go:
```diff
diff --git a/crate/src/lib.rs b/crate/src/lib.rs
index dc361c6..bc72b4a 100644
--- a/crate/src/lib.rs
+++ b/crate/src/lib.rs
@@ -7,7 +7,7 @@ use chrono::Utc;

 // The current version of the sandbox node we want to point to. This can be updated from
 // time to time, but probably should be close to when a release is made.
-const DEFAULT_SANDBOX_COMMIT_HASH: &str = "2c9375ee5ee307c2ce870c7dbd25eefd84fe8c36";
+const DEFAULT_SANDBOX_COMMIT_HASH: &str = "1.25.0/9b3d6ba551f561a028f0216051e031bc2ba0c6b7";

 const fn platform() -> &'static str {
     #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
@@ -29,7 +29,7 @@ fn local_addr(port: u16) -> String {

 fn bin_url(version: &str) -> String {
     format!(
-        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{}/master/{}/near-sandbox.tar.gz",
+        "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{}/{}/near-sandbox.tar.gz",
         platform(),
         version,
     )

```